src/test/e2e/helpers/database-init.ts

```diff
@@ -79,6 +79,7 @@ export interface ITestDb {
 export default async function init(
     databaseSchema: string = 'test',
     getLogger: LogProvider = noLoggerProvider,
+    configOverride: any = {},
```

Comment:
Would `configOverride: Partial<IUnleashOptions> = {}` work?

Reply:
Yeah, turns out that does work! Thanks, nice suggestion!